service/api_logic/news_logic.py

```diff
@@ -1,13 +1,12 @@
 import json
-from database.models import News
+from database.models import News, Sport
 from database.azure_blob_storage.save_get_blob import blob_get_news
 
 
 def get_news_by_count(count: int, session) -> str:
     news_records = session.query(News).order_by(News.save_at.desc()).limit(count).all()
     if not news_records:
         print(f"\033[31mNo news was found in the database.\033[0m")
```

Comment:
Consider creating separate method "print_text_in_red" (don't copy the method name. It's definitely not what you might want to have) which would handle the coloring of the text. 

Google DRY, YAGNI, KISS and, of course, SOLID.